etstool.py

```diff
@@ -82,8 +83,36 @@
 
 import click
 
+
+def current_platform():
+    """Return the current platform as a string."""
+    if sys.platform.startswith("win32"):
+        return "windows"
+    elif sys.platform.startswith("linux"):
+        return "linux"
+    elif sys.platform.startswith("darwin"):
+        if platform.machine() == "arm64":
```

Comment:
On apple silicon if an x86_64 Python binary is used then platform.machine() reports 'amd64'. You need to check `'RELEASE_ARM64' in platform.uname().version` to be more robust in detecting an apple silicon host.

Reply:
Thanks! Fixed in [ff55861](https://github.com/enthought/pyface/pull/1261/commits/ff55861adb1302360551e3e7a7facfaabe97a5f2).